nix/cabal-project.nix

```diff
@@ -0,0 +1,11 @@
+# This file is part of the IOGX template and is documented at the link below:
+# https://www.github.com/input-output-hk/iogx#33-nixcabal-projectnix
+
+{
+  modules = [{
+    packages.marlowe-symbolic.ghcOptions = [ "-Werror" ];
```

Comment:
Why do we have these here instead of in cabal.project? @hrajchert

Reply:
This is common to all repos. Basically we want -Werror in CI, but not locally

Reply:
OK. Might be nice to have some way to do this behind the scenes. E.g. ask haskell.nix "which packages in this package set were actually defined as top-level in the cabal.project?" and turn on Werror for all of those.